src/components/Block/Block.scss

```diff
@@ -1,48 +1,92 @@
+@use '~@strategies/tokens/color';
+@use '~@strategies/tokens/space';
+@use '~@strategies/tokens/elevation';
+
+@import '~@strategies/tokens/mixins';
 &__Block {
     position: absolute !important;
     display: flex;
-
+    overflow: hidden;
     cursor: grab;
-    opacity: .75;
+    // opacity: .75;
     pointer-events: all;
     transition: border-color .15s, opacity .15s;
+    border-color: color.$border-bold;
+    border-radius: space.$s100;
+    border-width: 1px;
+    border-style: solid;
 
     &:hover {
         opacity: 1;
         transition: border-color .15s, opacity .15s;
+        box-shadow: elevation.$shadow;
+        border-radius: space.$s100;
+        border-color: color.$border-active;
     }
 
     &--selected {
-        border-color: rgba($color_highlight, 1);
-        box-shadow: 0 0 3px 5px rgba($color_bg-inverse, .3);
+        box-shadow: elevation.$shadow;
         transition: border-color .15s, opacity .15s;
+        border-width: space.$s050;
+        border-radius: space.$s100;
+        border-color: color.$border-active;
     }
 
     &-content {
         flex: 1 1 auto;
+        
+        height: 100%;
+    }
 
+    
+
+    &-handle {
         height: 100%;
+        z-index: 2;
+        cursor: col-resize;
+        border-radius: space.$s100;
     }
 
-    &-label {
+    &-GroupLabel {
         position: absolute !important;
```

Comment:
Todo:  Go through the css in this repo and get rid of all the `!important`s. Gonna make a card from this.